packages/hemi-tunnel-actions/src/types.ts

```diff
@@ -0,0 +1,21 @@
+import { type Hash, TransactionReceipt } from 'viem'
+
+export type DepositEvents = {
+  'deposit-failed': [Error]
+  'deposit-failed-validation': [string]
+  'on-deposit-settled': []
```

Comment:
Events prefixed with `on-` might be renamed. The event handlers are usually prefixed that way but not the event names themselves.

Reply:
Events renamed here. [a346ac2](https://github.com/hemilabs/ui-monorepo/pull/1119/commits/a346ac2fe38d683aa2b5811488ce71ced0e16ef9)